Automattic/wp-calypso
# Earn page updates: adding Premium Content block card and updating cop…

#### Changes proposed in this Pull Request

This PR includes a variety of updates to the Earn page, including:

* Adding a card for the Premium Content block.
* Updating copy for the Simple Payments, Recurring Payments, and Referral program cards

Here's a BEFORE screenshot:
<img width="1381" alt="Screen Shot 2020-05-21 at 8 27 05 AM" src="https://user-images.githubusercontent.com/35781181/82603798-48a3c400-9b81-11ea-8f17-dd216f56bee9.png">

And here are two AFTER screenshots, the first is a Free plan user who is not eligible for the features and hasn't agreed to the referral program terms, and the second is a Business plan user who is eligible for everything and has agreed to the terms.

**Free plan user:**
<img width="1324" alt="Screen Shot 2020-05-21 at 4 18 09 PM" src="https://user-images.githubusercontent.com/35781181/82604017-9c161200-9b81-11ea-89b3-86ebf235626d.png">

**Business plan user:**
<img width="1295" alt="Screen Shot 2020-05-21 at 4 20 02 PM" src="https://user-images.githubusercontent.com/35781181/82603988-8f91b980-9b81-11ea-92aa-795cf6ca7a08.png">

#### Testing instructions
* Checkout branch (git fetch origin && git checkout -b update/earn-page-copy-updates origin/update/earn-page-copy-updates).
* Run Calypso (yarn start).
* Pay particular attention to the Premium Content card since that is a new addition.
* On a site with a Free plan, view the Earn page in the dashboard (Tools > Earn).
* Verify text updates are applied, that the primary CTAs lead to the Plans page, and that the learn more links point to the appropriate documentation. Click the terms link in the referral program card to confirm that the referral program terms are displayed. Click the button and confirm that the referral link appears.
* Login with a Business plan user and confirm that you have access to all features
* Click and verify that the “Unlock this feature” buttons route to /plans/[Site-Slug] (where [Site-Slug] is the slug of the site you have selected).
